exoctk/phase_constraint_overlap/phase_constraint_overlap.py

```diff
@@ -99,20 +99,20 @@ def phase_overlap_constraint(target_name, period=None, t0=None, obs_duration=Non
         maxphase : float
             The maximum phase constraint. '''
 
-    if obs_duration == None:
-        if period == None:
-            data = get_target_data(target_name)
-            
-            period = data['orbital_period']
-            transit_dur = data['transit_duration']
-            t0 = data['transit_time']
+    if period == None:
+        data, _ = get_target_data(target_name)
+        period = data['orbital_period']
 
+    if obs_duration == None:
+        data, _  = get_target_data(target_name)
+        transit_dur = data['transit_duration'] *24.0
         obs_duration = calculate_obsDur(transit_dur)
 
     minphase, maxphase = calculate_phase(period, obs_duration, window_size)
     
+    return minphase, maxphase
     # Is this the return that we want? Do we need to use t0 for something? 
```

Comment:
maybe remove the troubleshooting comments from the code @mfixstsci @catherine-martlin